src/cljs/athens/views/blocks.cljs

```diff
@@ -600,6 +603,7 @@
                        :generated-str nil
                        :old-string (:block/string block) ;; this is for detecting what's deleted to process page deletion
```

Comment:
Better names for strings

Reply:
`:atom-string` -> `:string/local`
`:generated-str` -> `:string/generated`
`:old-string` -> `:string/previous`